CMakeLists.txt

```diff
@@ -33,5 +37,20 @@ set(CMAKE_CUDA_EXTENSIONS OFF)
 # With CUDA language enabled above, this should find the toolkit alongside the compiler
 find_package(CUDAToolkit REQUIRED)
 
+#Find VecCore with correct backend
+set(VecCore_VERSION 0.5.2)
+set(VecCore_BACKEND CUDA)
+find_package(VecCore ${VecCore_VERSION} REQUIRED COMPONENTS ${VecCore_BACKEND})
+message(STATUS "Using VecCore version ${VecCore_VERSION}")
+set(ADEPT_LIBRARIES_EXTERNAL ${ADEPT_LIBRARIES_EXTERNAL} VecCore::VecCore)
+
+# Find VecGeom geometry headers library
+find_package(VecGeom REQUIRED)
+message(STATUS "Using VecGeom version ${VecGeom_VERSION}")
+# make sure we import VecGeom architecture flags
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${VECGEOM_CXX_FLAGS}")
+include_directories(${VECGEOM_INCLUDE_DIRS})
```

Comment:
This can be removed for VecGeom >= 1.1.7, which exports most usage requirements in the `VecGeom::vecgeom` target (so dependent on what we choose for min VecGeom version.

Reply:
Will do...